docs/stdlib/fields.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Fields Module | Juttle Language Reference
+---
+
+# Fields
+
+Reducers to manipulate fields within your data.
+
+## tail - reducer
+
+```
+... | reduce hosts = fields.tail(hostname, 5) | ...
+```
+
+Options    |                     Description                 | Required?
+---------- | ----------------------------------------------- | --------- :
+`field`    | the field name to store the values of in a list | Yes
+`length`   | the number of recent values to retain           | No, default: `1`
```

Comment:
I think `n` is a better name for this option.
- Does a better job matching '...containing the N last values...'.
- Thats what [R uses](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/head.html) (didn't check other languages).